src/fs.cpp

```diff
@@ -32,9 +32,9 @@ namespace filesystem {
                     std::istream_iterator<u8>(file),
                     std::istream_iterator<u8>());
 
-#ifndef NDEBUG
+    #ifndef NDEBUG
```

Comment:
remove indentation, that's the way almost all formatters do it
ditto in 3 occurences below

Reply:
literally didn't even do that, that's visual studio